src/lecture_feedback/lecture_feedback_facade.py

```diff
@@ -0,0 +1,58 @@
+import uuid
+
+import streamlit as st
+
+from lecture_feedback.application_state import ApplicationState
+from lecture_feedback.room import Room
+from lecture_feedback.session_state import SessionState
+from lecture_feedback.user_status import UserStatus
+
+
+class LectureFeedbackFacade:
+    def __init__(self) -> None:
+        self._application_state: ApplicationState = self._get_application_state()
+        self._session_state = SessionState()
+
+    @staticmethod
```

Comment:
```suggestion
        self._application_state: ApplicationState = LectureFeedbackFacade._get_application_state()
        self._session_state = SessionState()

    @classmethod
```

To be more specific with the singleton thing